cdhweb/pages/models.py

```diff
@@ -598,3 +599,15 @@ def clean_fields(self, exclude=None):
         # require end date to be greater than start date
         if self.start_date and self.end_date and not self.end_date >= self.start_date:
             raise ValidationError("End date must be after start date")
+
+
+@register_setting(icon="edit")
+class PurpleMode(BaseSiteSetting):
+    purple_mode = models.BooleanField(
+        default=False,
+        help_text="""This will turn the site purple 
+                                      by transforming all shades of cyan on the site into shades of purple""",
+    )
+
```

Comment:
Woo, tracked down a simpler icon fix (though marginally less easy to apply universally)
```suggestion
    panels = [FieldPanel("use_card_db", icon="placeholder")]
```

Reply:
@sarahframe I'll leave this with you :D

Reply:
yay thanks Haydn :D

Reply:
<img width="773" alt="image" src="https://github.com/springload/cdh-web/assets/110040326/2de769fd-9a5d-454b-806e-19f7c31977d7">